qcore/microtime.py

```diff
@@ -148,14 +159,24 @@ def format_utime_as_iso_8601(utime, *, drop_subseconds=False, tz=timezone.utc):
     """Get ISO 8601 Time string for the given microseconds time.
 
     Example output for the default UTC timezone:
-    "2022-10-28T23:37:38+00:00"
+    "2022-10-31T18:02:03.123456+00:00"
 
     """
     if drop_subseconds:
         utime = int(utime / SECOND) * SECOND
     return utime_as_datetime(utime, tz=tz).isoformat()
 
 
+def iso_8601_as_utime(iso_datetime: str):
```

Comment:
Is Cython OK with the type annotation here?

I believe Cython has some support for type annotations now, but that may be only in the eternally-in-alpha 3.0 branch.

Reply:
Oh, I didn't realize I added this. I didn't get any error from CY, but let me remove it for consistency.